manifests/windows/install.pp

```diff
@@ -32,7 +37,7 @@
     content => template('puppet_agent/install_puppet.bat.erb')
   }->
   exec { 'install_puppet.bat':
-    command   => "${::system32}\\cmd.exe /c start /b ${::system32}\\cmd.exe /c \"${env_temp_variable}\\install_puppet.bat\"",
+    command   => "${_cmd_location} /c start /b ${_cmd_location} /c \"${env_temp_variable}\\install_puppet.bat\"",
```

Comment:
I understand that it will work in that manner however the code reading is more confusing, it is more simplistic and with ${_cmd_location} in both places